src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -287,6 +313,8 @@ private void EnsureSignatureIsLoaded()
                     type = TypeWithAnnotations.Create(new PointerTypeSymbol(TypeWithAnnotations.Create(fixedElementType)));
                 }
 
+                ImmutableInterlocked.InterlockedInitialize(ref _lazyRefCustomModifiers, CSharpCustomModifier.Convert(fieldInfo.RefCustomModifiers));
```

Comment:
>_lazyRefCustomModifiers [](http://example.com/codeflow?start=63&length=23)

Do we have direct tests asserting where regular and ref modifiers go?

Reply:
Yes.